src/pages/admin/ExamReports.tsx

```
@@ -0,0 +1,207 @@
+import { Helmet } from 'react-helmet-async';
+
+import { PATH_ADMIN } from '../../constants/routes';
+import { Avatar, Button, Card, Modal, Table } from 'antd';
+import {
+  getOrganizationRequestService,
+  verifyOrganizationService,
+} from '../../api/services/Admin';
+import { useEffect, useState } from 'react';
+import { Organization } from '../authentication/forms/Organization';
+
+interface Organization{
```

Biome

[error] Shouldn't redeclare 'Organization'. Consider to delete it or rename it.

'Organization' is defined here:


(lint/suspicious/noRedeclare)